sklearn/tests/test_pipeline.py

```diff
@@ -142,6 +142,17 @@ def fit(self, X, y):
         return self
 
 
+class DummyEstimatorParams(BaseEstimator):
+    """Mock classifier that takes params on predict"""
+
+    def fit(self, X, y):
+        return self
```

Comment:
Looks like this line is not covered by any test, maybe it is enough to call `pipe.fit` in your test?

Reply:
@lesteve Done 👍